ethcore/src/engines/authority_round/mod.rs

```diff
@@ -626,7 +637,7 @@ trait AsMillis {
 
 impl AsMillis for Duration {
 	fn as_millis(&self) -> u64 {
-		self.as_secs()*1_000 + (self.subsec_nanos()/1_000_000) as u64
+		self.as_secs() * 1_000 + (self.subsec_nanos() / 1_000_000) as u64
```

Comment:
[Duration::as_millis](https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_millis) is available since rust 1.33, though it returns `u128`. Do we want to remove this trait? (Maybe out of scope of this PR)

Reply:
I prefer to keep it out this PR. If people agree with `this` approach it should be easy just to move the `AsMillis` trait to the `time_util library`. However, the implementation above truncates `Duration` into u64 which we don't want in the library.

Seems to be ok for the use case in `authority_round`